src/makeRobot.js

```diff
@@ -38,7 +38,86 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const Robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    goForward(number) {
+      if (!number) {
+        this.coords.y++;
+
+        return this;
+      } else if (number > 0) {
+        this.coords.y += number;
+
+        return this;
+      } else {
+        return this;
+      }
```

Comment:
```suggestion
    goForward(number = 1) {
      if (number > 0) {
        this.coords.y += number;
      }
      
      return this;
```
Consider setting the default value for number to optimize your code and make it a bit cleaner